README.md

```diff
@@ -1,8 +1,9 @@
-# StorPool Storage Driver
+# StorPool storage driver
 
-## Description
+[StorPool](https://storpool.com/) is a block storage software (software-defined storage solution). It runs on standard x86 servers, converting them into high-performance and highly available virtual (SAN) storage arrays. It can also run beside compute workloads in a hyper-converged manner.
 
-The [StorPool](https://storpool.com/) datastore driver enables OpenNebula to use a StorPool storage system for storing disk images.
+The StorPool datastore driver enables OpenNebula to use a StorPool storage system for storing disk images.
+The documentation of the driver is available at [StorPool's Knowledge Base](https://kb.storpool.com/storpool_integrations/OpenNebula/).
```

Comment:
For now on, any update to github will be auto-synced to the kb, so any change pushed in github will be updated in the kb too? otherwise I prefer the documentation of the driver to be pointed to github.